.werft/values.dev.yaml

```diff
@@ -35,9 +35,8 @@ components:
     makeNewUsersAdmin: true # for development
     theiaPluginsBucketName: gitpod-core-dev-plugins
     enableLocalApp: true
-    blockNewUsers: true
-    blockNewUsersPasslist:
-    - "gitpod.io"
+    blockNewUsers: false
```

Comment:
**reminder**: To remove this before merging! 😇

Reply:
Indeed, split out into https://github.com/gitpod-io/gitpod/pull/4211